lib/manageiq/providers/ibm_cloud/cloud_tools/activity_tracker.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'ibm_cloud_activity_tracker'
```

Comment:
Can you move this require down into the `sdk_client` method ?  That way we only will load the gem if someone is actively using it, this helps keep the core memory lower.